tests/backup/main.tf

```
@@ -0,0 +1,55 @@
+provider "aws" {
+  version = "~> 2.14"
+  region  = "us-west-2"
+}
+
+resource "random_string" "r_string" {
+  length  = 10
+  upper   = false
+  special = false
+}
+
+module "db" {
+  source = "git@github.com:rackspace-infrastructure-automation/aws-terraform-dynamo//?ref=v0.0.3"
```

I would pin this to master, so we don't have to keep up with changes for our tests

```suggestion
  source = "git@github.com:rackspace-infrastructure-automation/aws-terraform-dynamo//?ref=master"
```

---

is that what we do in other places?  If we pin to  master  we possibly introduce breaking  changes into the tests  when someone changes that module's interface. I personally don't see the big deal if we use older module versions  for modules not under test.

---

Its what we have done elsewhere.  Was more in an effort to make sure everything is staying compatible along the way and discover if a compatibility issue has come up.  But if it's a concern, can we bump to the latest version?

---

At the very least that . But if it’s what we are doing everywhere then I’ll accept that for what it is

---

More streamlined...